network-api/networkapi/settings.py

```diff
@@ -45,7 +45,7 @@
     DEBUG=(bool, False),
     DJANGO_LOG_LEVEL=(str, 'INFO'),
     DOMAIN_REDIRECT_MIDDLWARE_ENABLED=(bool, False),
-    ENABLE_PROFILE_DIRECTORY_BLOCK=(bool, False),
+    ENABLE_PROFILE_DIRECTORY_BLOCK=(bool, True),
```

Comment:
no, let's keep this `False`. We'll remove this flag in a follow-up https://github.com/mozilla/foundation.mozilla.org/issues/2706

Reply:
Done.